trade-finance-manager-ui/component-tests/utilisation-reports/_macros/premium-payments-table.component-test.ts

```diff
@@ -488,16 +583,15 @@ describe(component, () => {
     const [firstRowId, ...otherIds] = feeRecordIds;
 
     const firstRowSelector = `[data-cy*="premium-payments-table-row--feeRecordId-${firstRowId}"]`;
-    wrapper.expectElement(`${firstRowSelector} td:contains("${totalPaymentsReceived.formattedCurrencyAndAmount}")`).toExist();
-    wrapper.expectElement(`${firstRowSelector} td:contains("${totalPaymentsReceived.formattedCurrencyAndAmount}")`).hasClass(numericCellClass);
-    wrapper
-      .expectElement(`${firstRowSelector} td:contains("${totalPaymentsReceived.formattedCurrencyAndAmount}")`)
-      .toHaveAttribute('data-sort-value', totalPaymentsReceived.dataSortValue.toString());
+    const firstRowCellSelector = `${firstRowSelector} [data-cy="total-payments-received"]`;
+    wrapper.expectText(firstRowCellSelector).toRead(totalPaymentsReceived.formattedCurrencyAndAmount!);
+    wrapper.expectElement(firstRowCellSelector).hasClass(numericCellClass);
+    wrapper.expectElement(firstRowCellSelector).toHaveAttribute('data-sort-value', totalPaymentsReceived.dataSortValue.toString());
 
     otherIds.forEach((id) => {
       const rowSelector = `[data-cy*="premium-payments-table-row--feeRecordId-${id}"]`;
       wrapper.expectElement(`${rowSelector}`).toExist();
-      wrapper.expectElement(`${rowSelector} td:contains("${totalPaymentsReceived.formattedCurrencyAndAmount}")`).notToExist();
+      wrapper.expectText(`${rowSelector} [data-cy="total-payments-received"]`).toRead('');
```

Comment:
Could we split this test up into different files? Lots going on and now has 751 lines